tests/gdbstub-test/main.sh

```diff
@@ -0,0 +1,16 @@
+GREEN='\033[0;32m'
```

Comment:
Specify [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) to `#!/usr/bin/env bash` for the sake of bashism.